src/test/kotlin/org/malachite/estella/application/ApplicationsIntegration.kt

```diff
@@ -149,6 +154,61 @@ class ApplicationsIntegration : BaseIntegration() {
         expectThat(applications.size).isGreaterThanOrEqualTo(1)
     }
 
+    @Test
+    @Order(6)
+    fun `should be able to change stage to next`() {
+        val offer = getOffer()
+        val application = getOfferApplications(offer!!).find { it.jobSeeker.user.mail == getJobSeeker().user.mail }!!
+        expectThat(application.stage.type).isEqualTo(StageType.APPLIED)
+        val response = httpRequest(
+            path = "/api/applications/${application.id}/next",
+            method = HttpMethod.PUT,
+            headers = mapOf(EStellaHeaders.jwtToken to getAuthToken(hrPartner.user.mail, "a")),
+            body = mapOf(
+                "files" to setOf<JobSeekerFilePayload>()
+            )
+        )
```

Comment:
nie wiem czy nie wywaliłbym tego requestu do funkcji skoro używasz tego tutaj i w kolejnym teście.
Dodatkowo zamieniłbym to "a" na jakąś zmienną. I po co tutaj pliki?